GitHub.Collectors.Functions/GitHubFunctions.cs

```diff
@@ -652,6 +652,97 @@ public async Task Traffic([QueueTrigger("traffic")] string queueItem, ExecutionC
             }
         }
 
+        [FunctionName("PointCollector")]
+        public Task PointCollector([QueueTrigger("point")] string queueItem, ExecutionContext executionContext, ILogger logger, int dequeueCount)
```

Comment:
Let's call this pointcollector. I know the rest of the GH queue don't have the collector suffix, but that would make things more consistent with ADO.